events/code_review_events/workflow.py

```diff
@@ -277,24 +272,26 @@ def __init__(self, cache_root):
             self.webserver.register(self.bus)
 
             # Create pulse listener
-            exchanges = []
+            exchanges = {}
             if taskcluster_config.secrets["autoland_enabled"]:
                 logger.info("Autoland ingestion is enabled")
-                exchanges += [
-                    # autoland ingestion
+                # autoland ingestion
+                exchanges[QUEUE_PULSE_AUTOLAND] = [
                     (PULSE_TASK_GROUP_RESOLVED, ["#.gecko-level-3.#"])
                 ]
-            if publish:
-                # unit test failures
-                exchanges += [(PULSE_TASK_COMPLETED, ["*.*.gecko-level-3._"])]
+
+            # unit test failures
+            exchanges[QUEUE_PULSE_TRY_TASK_END] = [
+                (PULSE_TASK_COMPLETED, ["*.*.gecko-level-3._"])
+            ]
```

Comment:
All these lines can be removed too (they were part of the bad rebase).

Reply:
done